nohup: ignoring input
 * Serving Flask app 'Web_UI/app.py'
 * Debug mode: off
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on http://127.0.0.1:5436
Press CTRL+C to quit
127.0.0.1 - - [14/May/2024 12:46:38] "GET / HTTP/1.1" 302 -
[2024-05-14 12:46:38,031] ERROR in app: Exception on /printer_status [GET]
Traceback (most recent call last):
  File "/home/cjtorralba/.local/lib/python3.10/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/cjtorralba/.local/lib/python3.10/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/cjtorralba/.local/lib/python3.10/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/cjtorralba/.local/lib/python3.10/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/cjtorralba/PycharmProjects/FUME/Web_UI/app.py", line 18, in status
    with open('./data/data.json') as json_file, open('./data/printers.json') as printer_file:
FileNotFoundError: [Errno 2] No such file or directory: './data/data.json'
127.0.0.1 - - [14/May/2024 12:46:38] "GET /printer_status HTTP/1.1" 500 -
127.0.0.1 - - [14/May/2024 12:46:38] "GET /favicon.ico HTTP/1.1" 404 -
